src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,28 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String printDiamond(int diamondHeight) {
+		if (diamondHeight <= 0 || diamondHeight % 2 == 0) {
 			return null;
 		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
-			builder.insert(0, chars);
-			builder.append(chars);
+		StringBuilder diamondBuilder = new StringBuilder(new String(buildDiamondRow(diamondHeight, diamondHeight)));
+		for (int diamondWidth = diamondHeight - 2; diamondWidth > 0; diamondWidth -= 2) {
```

Comment:
`diamondWidth`  sounds like it is a property of the diamond. Rather, it is a property of the row, so it might be preferable to use something like `rowWidth`.

Reply:
yes youre right